src/main/resources/the_warlordResources/localization/eng/WarlordMod-Power-Strings.json

```diff
@@ -115,7 +120,7 @@
   "the_warlord:FocusPunchPower": {
     "NAME": "Focus Punch",
     "DESCRIPTIONS": [
-      "At the start of next turn, deal #b%1$s damage. NL If you take damage, gain #b5 #yTension."
+      "At the start of next turn, deal #b%1$s damage. NL If you take damage, gain #b2 #yTension."
```

Comment:
As suggested in FocusPunchPower, we should use amount2 for the Tension amount, so that the tension stacks when you play multiple of these.